src/receivers/verify-redirect-opts.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Helper to verify redirect uri and redirect uri path exist and are consistent
+ * when supplied.
+*/
+import { AppInitializationError } from '../errors';
+import { HTTPReceiverOptions, HTTPReceiverInstallerOptions } from './HTTPReceiver';
+
+export interface RedirectOptions {
+  redirectUri?: HTTPReceiverOptions['redirectUri'],
+  redirectUriPath?: HTTPReceiverInstallerOptions['redirectUriPath'],
+}
+
+export function verifyRedirectOpts({ redirectUri, redirectUriPath }: RedirectOptions): void {
```

Comment:
I added a comment on the corresponding test code side but missing only redirectUri should be okay.